build.gradle

```diff
@@ -215,17 +215,13 @@ dependencies {
     api 'io.github.microutils:kotlin-logging:2.1.21'
 
     implementation 'org.apache.logging.log4j:log4j-slf4j2-impl'
-    implementation 'org.apache.logging.log4j:log4j-1.2-api'
     implementation 'org.apache.logging.log4j:log4j-core'
 
     implementation 'io.prometheus:simpleclient'
     implementation 'io.prometheus:simpleclient_hotspot'
     implementation 'io.prometheus:simpleclient_httpserver'
     implementation 'io.prometheus:simpleclient_log4j2'
 
-    implementation ('com.squareup.okhttp3:okhttp:4.10.0') {
-        because ('fix vulnerability in transitive dependency <kubernetes-client>')
-    }
```

Comment:
Why have you removed that dependency?

Reply:
After removing log4j:1.x version the client starts to throw NoSuchMethod exception related to change in a log method signature. Unfortunately I can't reproduce the same problem now and revert my changes